src/voxel.h

```diff
@@ -569,6 +580,8 @@ class VoxelManipulator /*: public NodeContainer*/
 	*/
 	u8 *m_flags;
 
+	static const MapNode ContentIgnoreNode;
```

Comment:
* Functions

Couldn't you optimize one of the already existing functions instead of adding a new?